### DISCUSSION TITLE
what's left for the next release?

### DISCUSSION THREAD

**Discussion by Rexogamer** [LibChecker/LibChecker#1282] [Q&A] [answered]
heya! thank you for the incredibly useful app :3

the last update on Google Play was in December, and it looks like a bunch has happened since. is there anything in particular left to do before the next release?

absolutely no rush, and thank you again for all the hard work you've done!

**Comment by zhaobozhen** [comment, marked as answer]
> heya! thank you for the incredibly useful app :3
> 
> 
> 
> the last update on Google Play was in December, and it looks like a bunch has happened since. is there anything in particular left to do before the next release?
> 
> 
> 
> absolutely no rush, and thank you again for all the hard work you've done!

Hi Sophie,

Everything is progressing, albeit a bit slowly. The remaining tasks before the next official release include a few feature requests (FRs) and the significant workload of multilingualizing the library rules. Once these tasks are completed, I will release the official version of 2.5.0. I hope to release the official version of LibChecker simultaneously with the official release of Android 15, if I have enough free time. :)
